generate-keys.sh

```diff
@@ -0,0 +1,7 @@
+# SPDX-FileCopyrightText: 2025 INFO.nl
+# SPDX-License-Identifier: EUPL-1.2+
+
+openssl req -new -x509 -days 9999 -config ./ca.example.cnf -keyout ./office-backend/ca-key.pem -out ./office-backend/ca-cert.pem
```

Comment:
If we need our own certificate it should be added to this repo? We are not going to ask everyone to generate their own certificate. That is undoable.

Reply:
I would add a brief code comment in this file just explaining what is done.